config/locales/modules/module-4.yml

```diff
@@ -0,0 +1,1643 @@
+          - explain the impact of language skills on learning and behaviour
+          - explain the importance of a language rich approach where interactions are promoted 
+
+          The early years foundation stage (EYFS) puts language at the heart of learning and development as it underpins all seven areas of development. The quality of the interactions and opportunities that children experience in the early years will form the foundations to promote success in later life. Children will thrive in an environment that is rich with opportunities for: 
+
+          - story telling
+          - singing
+          - back and forth interactions
+          - imaginative and creative play
+
+          Children’s language experiences outside of your setting will vary and it is also important to remember that children will all develop at their own individual pace. As an early years practitioner, you can help children to learn, rehearse and use a rich range of language with confidence. You can provide children with a range of opportunities to do this through:
+
+          - the activities you offer
+          - the environment you provide 
+          - being a good communications role model
+
+      4-1: 
+        heading: How children develop language and communication skills  
+        body: | 
+          The first 3 years of a child’s life are the most important for language and communication development.
+
+          All children will develop at their own individual rate. If you understand expected development, you will be able to support each child’s journey.
+
+          The first section of this module includes:
+
+          - stages of language development
+          - theories of language development   
+          - factors affecting language development
+
+      4-1-1: 
+        heading: Stages of language development
+        body: |
+          Babies and children move through typical stages while they are developing their language skills. Understanding these typical stages will help you identify speech and language problems early.
+
+          The developmental stages outlined in this training are only intended as a guide. It is important to remember that all babies and children will develop at their own pace. Not every child will reach these stages at the age specified in this course. As a practitioner, you should consider each child's individual circumstances to offer the right support. 
+
+          This topic includes:
+
+          <ul class="govuk-list govuk-list--bullet">
+              <li>the pre-linguistic stage:
+
+              <ul>
+                <li>sounds and noises before talking</li>
+                <li>effective practice</li>
+              </ul>
+              <br>
+              <li>the linguistic stage:
+              <ul>
+              <li>one and two word sentences</li>  
+              <li>meaningful sentences</li>
+              <li>fluent speech</li>
+              
```

Comment:
Hey, just spotted I think there should be some closing `<ul>` tags in here somewhere